### PR TITLE
feat(sql-editor): add the sql-editor section

### PR DESCRIPTION
**Pull request by xiaoluoboding**

### CODE REVIEW

File: pages/index.vue [hunk 1]
```diff
@@ -301,7 +303,218 @@
...
+          <div
+            class="px-4 max-w-xl mx-auto sm:px-6 lg:py-24 lg:max-w-none lg:mx-0 lg:px-0"
+          >
+            <div>
+              <div>
+                <span
+                  class="h-12 w-12 rounded-md flex items-center justify-center bg-indigo-600"
+                >
+                  <!-- Heroicon name: outline/code -->
+                  <svg class="w-6 h-6 text-white" width="32" height="32" preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24"><g fill="none"><path d="M5 12h14M5 12a2 2 0 0 1-2-2V6a2 2 0 0 1 2-2h14a2 2 0 0 1 2 2v4a2 2 0 0 1-2 2M5 12a2 2 0 0 0-2 2v4a2 2 0 0 0 2 2h14a2 2 0 0 0 2-2v-4a2 2 0 0 0-2-2m-2-4h.01M17 16h.01" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"></path></g></svg>
```
Comment (tianzhou) [inline pages/index.vue:335]: Let's remove the icon as it's getting really hard to find meaningful icons....

**Review by tianzhou — CHANGES_REQUESTED** [review]
Also to improve screenshot quality, let's construct real world example, like the product example on the top.

Currently, the screenshot shows some test name data, which is a bit lame.

File: pages/index.vue [hunk 2]
```diff
@@ -301,7 +303,218 @@
...
+                </span>
+              </div>
+              <div class="mt-6">
+                <h2
+                  class="text-3xl font-extrabold tracking-tight text-gray-900"
+                >
+                  Explore, Alter the Schema
+                </h2>
+                <h3 class="mt-4 text-lg text-gray-500">
+                  Easily explore the database connection with Project-based access control, and you can select table to see the schema information.
```
[the remark this answers is not in the document]
Comment (xiaoluoboding) [reply pages/index.vue:345]: OK. remove alter schema for now.

File: pages/index.vue [hunk 3]
```diff
@@ -301,7 +303,227 @@
...
+    <!-- begin/SQL Editor Section -->
+    <div class="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
+      <div class="lg:text-center">
+        <h2
+          class="mt-2 text-3xl sm:text-5xl lg:text-6xl leading-8 font-extrabold tracking-tight text-gray-900"
+        >
+          Simple, yet powerful SQL editor
+        </h2>
+        <h3 class="mt-4 max-w-3xl text-center text-xl text-gray-500 lg:mx-auto">
+          Easy-to-use query interface to SELECT database records.
```
Comment (tianzhou) [inline pages/index.vue:315]:
```suggestion
          Inspect records, explore schema, sharing SQL scripts, and more.
```

File: pages/index.vue [hunk 4]
```diff
@@ -301,7 +303,227 @@
...
+                </span>
+              </div>
+              <div class="mt-6">
+                <h2
+                  class="text-3xl font-extrabold tracking-tight text-gray-900"
+                >
+                  Explore the Schema
+                </h2>
+                <h3 class="mt-4 text-lg text-gray-500">
+                  Easily explore the database connection with Project-based access control, and you can select a table to see its schema.
```
Comment (tianzhou) [inline pages/index.vue:352]:
```suggestion
                  Explore the database record and schema guarded by the project-level access control.
```

File: pages/index.vue [hunk 5]
```diff
@@ -301,7 +303,227 @@
...
+                      d="M10 20l4-16m4 4l4 4-4 4M6 16l-4-4 4-4"
+                    ></path>
+                  </svg>
+                </span>
+              </div>
+              <div class="mt-6">
+                <h2
+                  class="text-3xl font-extrabold tracking-tight text-gray-900"
+                >
+                  Write queries effectively
```
Comment (tianzhou) [inline pages/index.vue:408]:
```suggestion
                  Smooth query editing experience
```

File: pages/index.vue [hunk 6]
```diff
@@ -301,7 +303,227 @@
...
+                </h2>
+                <h3 class="mt-4 text-lg text-gray-500">
+                  Enjoy the powerful built-in text editor based on Monaco Editor, with autocomplete suggestions, format SQL, shortcuts, and more.
```
Comment (tianzhou) [inline pages/index.vue:411]:
```suggestion
                  Autocomplete suggestions, SQL format, shortcuts, and more.
```

File: pages/index.vue [hunk 7]
```diff
@@ -301,7 +303,227 @@
...
+                >
+                  <!-- Heroicon name: outline/document-text -->
+                  <svg role="img" class="iconify iconify--heroicons-outline" width="32" height="32" preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24"><g fill="none"><path d="M9 12h6m-6 4h6m2 5H7a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h5.586a1 1 0 0 1 .707.293l5.414 5.414a1 1 0 0 1 .293.707V19a2 2 0 0 1-2 2z" stroke="#fff" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"></path></g></svg>
+                </span>
+              </div>
+              <div class="mt-6">
+                <h2
+                  class="text-3xl font-extrabold tracking-tight text-gray-900"
+                >
+                  Never miss your sheets
```
Comment (tianzhou) [inline pages/index.vue:455]:
```suggestion
                  Never miss your work
```

File: pages/index.vue [hunk 8]
```diff
@@ -301,7 +303,227 @@
...
+                </h2>
+                <h3 class="mt-4 text-lg text-gray-500">
+                  The sheets allows you to easily find them and reuse it later by saving them, and share them with your teammates.
```
Comment (tianzhou) [inline pages/index.vue:458]:
```suggestion
                  Save your favorite sheets and auto-record the execution history.
```

File: pages/index.vue [hunk 9]
```diff
@@ -301,7 +303,227 @@
...
+                </span>
+              </div>
+              <div class="mt-6">
+                <h2
+                  class="text-3xl font-extrabold tracking-tight text-gray-900"
+                >
+                  Share sheet with teammates
+                </h2>
+                <h3 class="mt-4 text-lg text-gray-500">
+                  Share the sheet as a link on the SQL statement with your teammates, with sharing options easily manage the sheet permissions.
```
Comment (tianzhou) [inline pages/index.vue:500]:
```suggestion
                  Share the sheet link with your teammates, with configurable access levels.
```

File: pages/index.vue [hunk 10]
```diff
@@ -301,7 +303,227 @@
...
+                  The sheets allows you to easily find them and reuse it later by saving them, and share them with your teammates.
+                </h3>
+              </div>
+            </div>
+          </div>
+          <div class="mt-12 sm:mt-16 lg:mt-0">
+            <div
+              class="pl-4 -mr-48 sm:pl-6 md:-mr-16 lg:px-0 lg:m-0 lg:relative lg:h-full"
+            >
+              <img
```
Comment (tianzhou) [inline pages/index.vue:467]: Ideally, overlay a history tab screenshot here as well.

**Comment by xiaoluoboding** [comment]
PTAL

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**